libraries/src/Document/HtmlDocument.php

```diff
@@ -528,7 +528,9 @@ public function getBuffer($type = null, $name = null, $attribs = [])
             return parent::$_buffer;
         }
 
-        $title = $attribs['title'] ?? null;
+        $type  = (string) $type;
+        $name  = (string) $name;
```

Comment:
```suggestion
        $name  ??= '';
```
`$type` could not be null at this stage (there is an if check before that), so cast `$type` to string is redundant and should be removed. `$name` could only be null or a string, so I think change the code to `$name  ??= '';` would make it more clear.